.github/composite-actions/run-jdbc-db-collation-tests/action.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Run JDBC Tests with database level collation'
+description: 'Run Babel JDBC test framework with database level collation'
+
+inputs:
+  input_dir:
+    description: 'Test input directory'
+    required: no
+    default: input
+
+runs:
+  using: "composite"
+  steps:
+    - name: Run JDBC Tests with db level collation
+      run: |
+          export PATH=/opt/mssql-tools/bin:$PATH
+          # sqlcmd -S localhost -U "jdbc_user" -P 12345678 -i .github/scripts/db_collation.sql
+          export PATH=~/${{env.INSTALL_DIR}}/bin:$PATH
+          export PG_SRC=~/work/babelfish_extensions/postgresql_modified_for_babelfish
+          export inputFilesPath=${{inputs.input_dir}}
+          cd test/JDBC/
+          sudo ~/psql/bin/psql -v ON_ERROR_STOP=1 -d jdbc_testdb -U runner -c "update sys.babelfish_sysdatabases set default_collation = 'bbf_unicode_cp1_ci_ai' where name = 'master';"
+          sudo ~/psql/bin/psql -v ON_ERROR_STOP=1 -d jdbc_testdb -U runner -c "select default_collation from sys.babelfish_sysdatabases where name = 'master';"
```

Comment:
Kindly create `install_dir` input variable and use it as follows instead of following statements
- `export PATH=~/${{env.INSTALL_DIR}}/bin:$PATH` -> `export PATH=~/${{inputs.install_dir}}/bin:$PATH`
- `~/psql/bin/psql` -> `~/${{inputs.install_dir}}/bin/psql`

Reply:
Same as above